### PR TITLE
Updates strings to ensure Windows and Linux/MacOS are included; aligns help text

### PR DESCRIPTION
**Pull request by annegentle**

### CODE REVIEW

**Comment by annegentle** [comment]
[rfr]

**Comment by annegentle** [comment]
@jrperritt can this be merged now that the release is out?

File: main.go [hunk 1]
```diff
@@ -37,35 +37,36 @@ func main() {
...
-			Name: "init",
-			Usage: strings.Join([]string{"For Linux and OS X, creates the `rack` man page and sets up command completion for the Bash shell.",
-				"\tFor Windows, creates a `posh_autocomplete.ps1` file in the `$HOME/.rack` directory. You must run the file to set up command completion."}, "\n"),
+			Name:   "configure",
+			Usage:  "Interactively create a config file for Rackspace authentication.",
+			Action: configure,
+		},
+		{
+			Name:  "init",
+			Usage: strings.Join([]string{"Interactively create a config file.",
```
Comment (jrperritt) [inline main.go]: Copy/paste relic? Regardless, that first string ^ should be removed.

**Comment by jrperritt** [comment]
After that ^ gets fixed, this'll be ready to merge.

**Comment by annegentle** [comment]
Got it! Good catch, will fix.

**Comment by annegentle** [comment]
Ok, ready for review.

File: docs/globaloptions.rst [hunk 2]
```diff
@@ -8,7 +8,12 @@ and allow you to customize various aspects of the ``rack`` tool at runtime.
 These options may override configuration file or environment variables you have
 set previously, change output format or other aspects of the tool.
 
-For example:
+Display any available command-line flags by appending `--help` to list the
+flags available::
+
+    rack <service> <command> <subcommand> --help
```
Comment (jrperritt) [inline docs/globaloptions.rst]: I think we decided on `rack <service> <subservice> <action> --help` for describing the command format.

File: docs/globaloptions.rst [hunk 3]
```diff
@@ -201,7 +206,7 @@ And it is available per command::
        help, h	Shows a list of commands or help for one command
 
 
-And again, per subcommand:
+And again, per subcommand::
```
Comment (jrperritt) [inline docs/globaloptions.rst]: s/subcommand/subservice for consistency with the change above ^

**Comment by annegentle** [comment]
@jrperritt ready for review; sure hope I got it all!

**Comment by jrperritt** [comment]
Looks good. +2

**Pull request merged**